notebooks/255-bark-text-to-audio/255-bark-text-to-audio.ipynb

```diff
@@ -0,0 +1,1143 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.    Bark is a universal text-to-audio model created by [Suno](www.suno.ai). \
</pre><p>This URL points to local address. Please prepend http:// or https://.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>